etc/config-template.toml

```diff
@@ -56,9 +56,12 @@
 # max recursion level allowed when decoding dag expression
 # end-point-recursion-limit = 1000
 
+# max time to handle coprocessor request before timeout
+# end-point-request-max-handle-secs = 60
```

Comment:
Please add this configuration to another PR and use a `ReadableDuration` for it.